base/strings/util.jl

```diff
@@ -137,13 +137,15 @@ end
 const _default_delims = [' ','\t','\n','\v','\f','\r']
 
 """
-    lstrip(s::AbstractString[, chars::Chars])
+    lstrip(str::AbstractString[, chars])
 
-Return `s` with any leading whitespace and delimiters removed.
-The default delimiters to remove are `' '`, `\\t`, `\\n`, `\\v`,
-`\\f`, and `\\r`.
-If `chars` (a character, or vector or set of characters) is provided,
-instead remove characters contained in it.
+Remove leading characters from `str`.
+
+The default behaviour is to remove leading whitespace and delimiters: see
+[`isspace`](@ref) for precise details.
+
+The optional `chars` argument specifies which characters to remove: it can be a single character,
+vector or set of characters, or a predicate function.
```

Comment:
The predicate function should always go first per our conventions, documented in the style guide in the manual.

Reply:
It is a "general rule" not a "must". Having the function argument second here seems much more natural to me.

Reply:
![guidelines](https://user-images.githubusercontent.com/11698744/40707756-e915b854-63f1-11e8-8802-5896de6af71b.jpg)

Reply:
I disagree wholeheartedly; it seems very unnatural to me.